CMakeLists.txt

```diff
@@ -76,9 +76,13 @@ install(
 
 # make find_package(gridformat) work when used from build directory
 if (GRIDFORMAT_HIGHFIVE_SOURCE_INCLUDED)
+    configure_file(
```

Comment:
So I had to manually make high five config file available in the build tree... FetchContent_MakeAvailable didn't automatically produce it for some reason...